src/Approvals/SFA.DAS.Approvals/InnerApi/Responses/BulkUploadAddLogResponse.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace SFA.DAS.Approvals.InnerApi.Responses
+{
+    public class BulkUploadAddLogResponse
+    {
+        public BulkUploadAddLogResponse() { }
+
```

Comment:
The convention is we usually don't add ctors if they are not doing anything

Reply:
Removed